saleor/graphql/account/bulk_mutations/customer_bulk_update.py

```diff
@@ -556,8 +557,16 @@ def post_save_actions(cls, info, manager, instances, old_instances):
         app = get_app_promise(info.context).get()
         staff_user = info.context.user
         users_with_name_or_email_updated = []
+        webhooks_meta = get_webhooks_for_event(
+            WebhookEventAsyncType.CUSTOMER_METADATA_UPDATED
+        )
```

Comment:
Can we move it to line 632 to call it only if needed?

Reply:
moving this to line 632 may result in using this query multiple times cause it will be in `for` loop - I have no better idea for now - but maybe getting all webhooks for both events at once, then filtering them might be a better approach (this might be a next step improvement)

Reply:
As a next step we could make `get_webhooks_for_event` a dataloader where webhook event names would be the keys.

Reply:
@Air-t Maybe open a task for this, and let's add it to Q4 roadmap so we could try to tackle it later.